src/pages/press.tsx

```diff
@@ -0,0 +1,235 @@
+                                        />
+                                        <path className="cls-5" d="M624.68,60.21l0,37.84C618.85,90.9,610.26,87,600.32,87c-18,0-31.56,14.09-31.55,32.75,0,18.5,13.6,32.45,31.6,32.43,10,0,18.57-3.88,24.37-11v9.42h7.44l-.06-90.41Zm0,48.28v22.28a26,26,0,0,1-23.09,14.47c-14.41,0-25.28-11-25.29-25.51,0-14.7,10.85-25.79,25.25-25.8A26.34,26.34,0,0,1,624.72,108.49Z"
+                                        />
+                                    </svg>
+                                </a>
+                            </div>
+                            <span>Logo for dark backgrounds</span>
+                        </div>
+                        <div className="logo">
+                            <div className="svg-container">
+                                <a download href={Logo}>
+                                    <svg id="logo-final" xmlns="http://www.w3.org/2000/svg" style={{ transform: 'scale(2.6) translateY(15px)' }} viewBox="0 0 178.7 206.4">
+                                        <polygon className="st0" fill="#0087BE" points="89.3,185.6 17.7,144.6 17.7,61.9 17.8,61.9 0,51.6 0,154.8 0,154.8 89.3,206.4 89.3,206.4"
+                                        />
+                                        <polygon className="st0" fill="#0087BE" points="89.3,103.2 89.3,103.2 35.8,72.3 35.7,72.3 35.7,134.2 89.3,164.9 89.3,103.2"
+                                        />
+                                        <polygon className="st1" fill="#1AA6E4" points="178.7,51.6 89.3,0 0,51.6 0,51.6 17.8,61.9 89.3,20.6 161,61.8 178.7,51.6 178.7,51.6"
+                                        />
+                                        <polygon className="st1" fill="#1AA6E4" points="143,72.2 89.4,41.3 35.8,72.3 89.3,103.2"
+                                        />
+                                        <polygon className="st2" fill="#0092CF" points="89.3,103.2 89.3,103.2 89.3,103.2 89.3,103.2 89.3,103.2"
+                                        />
+                                        <polygon className="st2" fill="#0092CF" points="89.3,103.2 89.3,103.2 89.3,103.2 143,72.2 143,72.2"
+                                        />
+                                        <polygon className="st2" fill="#0092CF" points="161,61.8 161,61.8 178.7,51.6 178.7,51.6"
+                                        />
+                                        <polygon className="st2" fill="#0092CF" points="178.7,51.6 161,61.8 143,72.2 89.3,103.2 89.3,103.2 89.3,103.2 89.3,103.2 89.3,112.4 89.3,112.4 89.3,136.9 89.3,136.9 89.3,164.9 89.3,164.9 143,134.1 143,113.7 107,134.2 107,113.4 161,82.7 161,144.6 89.3,185.6 89.3,206.4 178.7,154.8 178.7,154.8 178.7,154.8"
+                                        />
+                                    </svg>
+                                </a>
+                            </div>
+                            <span>Figurative logos</span>
```

Comment:
I only see one logo here. Also, I don't quite understand "figurative" here. Here is an alternative suggestion:

```suggestion
                            <span>Logo without text</span>
```